src/rust/engine/Cargo.toml

```diff
@@ -141,3 +141,9 @@ workunit_store = { path = "workunit_store" }
 testutil = { path = "./testutil" }
 fs = { path = "./fs" }
 env_logger = "0.5.4"
+
+[patch.crates-io]
+# Patch all transitive uses of Prost to refer to the version with `Bytes` support.
```

Comment:
Would be good to reference the particular PR you're waiting on.

Reply:
Added link.